uint32 seq
time stamp

uint8 NOT_INITIALIZED=0
uint8 MASK_FINGER_READY=1
uint8 MASK_GPIO_READY=2
uint8 MASK_ERROR=4
uint8 status

uint8 RC_OK=0
uint8 rc

uint32 pins

JointValues joints
